be/src/storage/lake/delta_writer_finish_mode.h

```diff
@@ -19,6 +19,7 @@ namespace starrocks::lake {
 enum DeltaWriterFinishMode {
     kWriteTxnLog,
     kDontWriteTxnLog,
+    kSkipTxnLog,
```

Comment:
what's the difference between skipTxnLog and DontWriteTxnLog?

Reply:
`DontWriteTxnLog` will create TxnLog and put it in metacache, without put it into S3.

Reply:
And `skipTxnLog` will do nothing, no create TxnLog.